### ISSUE TITLE
Any way to avoid loading FxParallax assets?

### ISSUE THREAD

**Issue by phaberest** [zerodevx/svelte-img#27]
I see that the css for FxParallax is adding an item to my request chain and since I am not using any of its features in this project I would like to disable it and just keep the Img side, is there any way to do it?

I imported as `import Img from '@zerodevx/svelte-img'` everywhere

**Comment by zerodevx** [comment]
I don't quite understand your question tbh?

```html
<script>
  import Img, { FxParallax } from '@zerodevx/svelte-img'
  import src from '$lib/a/cat.jpg?as=run'
</script>

<!-- Img component -->
<Img {src} />

<!-- Img component with Parallax -->
<FxParallax {src} />
```

**Comment by phaberest** [comment]
Thank you @zerodevx, that's exactly what I mean. I never used it anywhere in the project, but it's getting loaded in the final build anyway

**Comment by zerodevx** [comment]
This should be fixed with `v2.0.2`. Thanks for bringing this to attention - great catch! 👍
